decidim-participatory_processes/app/views/decidim/participatory_processes/pages/home/_highlighted_processes.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="row">
+  <h3 class="section-heading"><%= t(".active_processes") %></h3>
+  <div class="row collapse">
+    <div class="row small-up-1 smallmedium-up-2 mediumlarge-up-3
+      large-up-4 card-grid">
+      <% highlighted_participatory_processes.each do |process| %>
```

Comment:
This variable should be taken away from the controller and moved somewhere else inside `participatory_processes`.